File: 2021/2/druga.py
def main():
    input = open("input.txt")
    horizontal = 0
    depth = 0
    aim = 0
    for line in input:
        besede = line.split()
        print(besede)
        if (besede[0] == "forward"):
            horizontal += int(besede[1])
            depth += aim * int(besede[1])
        else:
            if (besede[0] == "down"):
                aim += int(besede[1])
            else:
                aim -= int(besede[1])
    input.close()
    print(horizontal * depth)
main()
# 1813062561

File: 2021/2/druga.c
#include <stdio.h>
#include <string.h>

int main() {
    FILE* input = fopen("input.txt", "r");
    int num;
    char* word;
    int horizontal = 0, depth = 0, aim = 0; 
    while (fscanf(input, "%s %d", word, &num) != EOF) {
            if (strcmp(word, "forward") == 0) {
                horizontal += num;
                depth += aim * num;
            }
            else if (strcmp(word, "down") == 0) 
                aim += num;
            else 
                aim -= num;
    }
    fclose(input);
    printf("%d\n", horizontal * depth);
}